frontend/packages/ceph-storage-plugin/src/components/namespace-store/namespace-store-form.tsx

```diff
@@ -83,7 +89,7 @@ const NamespaceStoreForm: React.FC<NamespaceStoreFormProps> = withHandlePromise<
         ssl: false,
       },
     };
-    if (externalProviders.includes(provider)) {
+    if (provider !== BC_PROVIDERS.FILESYSTEM && externalProviders.includes(provider)) {
```

Comment:
Why is this required? Is `FileSystem` an external provider? Could we perhaps update `getExternalProviders`?

Reply:
I don't want those specs for Filesystem type. No buckets/secrets for filesystem type as they use native PVCs

Reply:
then remove `FileSystem` from `externalProviders`. I don't see the need to have two conditions when this logic can be added somewhere else.

Reply:
removed